frontend/src/pages/shipments/ShipmentDetails.tsx

```diff
@@ -0,0 +1,86 @@
+import { FunctionComponent } from 'react'
+import Badge from '../../components/Badge'
+import ReadOnlyField from '../../components/forms/ReadOnlyField'
+import Spinner from '../../components/Spinner'
+import { useShipmentQuery } from '../../types/api-types'
+import {
+  formatCountryCodeToName,
+  formatLabelMonth,
+  getShipmentStatusBadgeColor,
+} from '../../utils/format'
+
+interface Props {
+  shipmentId: number
+}
+
+const ShipmentDetails: FunctionComponent<Props> = ({ shipmentId }) => {
```

Comment:
This area already existing but it was moved into a tab. Here's what it looks like:

<img width="335" alt="Screen Shot 2021-03-19 at 3 45 46 PM" src="https://user-images.githubusercontent.com/3411183/111849191-3e01aa80-88ca-11eb-9c8d-8ebc2ace61fd.png">

Reply:
though (non-blocking): maybe what we want is this view for the admin, and for sending group captains a view without the offers tab and instead a button to create an offer or a summary of the offer they've created with links to edit.

Reply:
I didn't realize that group captain could only create a single offer per shipment. I totally agree with your thought then, let's simplify this for non-admins in a future PR.